index.bs

```diff
@@ -1036,6 +1037,10 @@ An <dfn>attribution rate-limit record</dfn> is a [=struct=] with the following i
 :: A [=moment=].
 : <dfn>expiry time</dfn>
 :: Null or a [=moment=].
+: <dfn>entity ID</dfn>
+:: An [=event-level report=]'s [=event-level report/report ID=] or an
+    [=aggregatable report=]'s [=aggregatable report/report ID=] or an
```

Comment:
What is the purpose in storing the aggregatable report ID or source ID? IIUC the ID here is only needed so that rows corresponding event-level attributions can be replaced.

Reply:
Source ID will be used for some upcoming features where we want to modify some rate-limit data.

Aggregate Report id is not used right but might have utility if we have deletion cases for aggregate report.

Reply:
I'd leave unused fields out unless there's a compelling reason to include them.

Reply:
+1 to @giladbarkan-github. The spec can be modified in the future if there's a reason to store other IDs here. We don't need to add support for speculative extension, unless you're planning to post a specific proposal in the near future, in which case, maybe add a non-normative `Note` here explaining that this will be used for a particular purpose.

Reply:
The IDs for sources and aggregatable reports will be used in https://github.com/WICG/attribution-reporting-api/pull/1217, but feel free to remove this from this PR and I can update there.